case_studies.md

```diff
@@ -0,0 +1,30 @@
+# Turing Way - case studies for reproducible research
+
+Reproducible research is work that can be independently verified. 
+In practice, it means sharing the data and code that were used to generate published results - yet this is often easier said than done.
+The Turing Way is a guide to reproducible data science that will support students and academics as they develop their code, with the aim of helping them produce work that will be regarded as gold-standard examples of trustworthy and reusable research. 
+It is openly developed and any and all questions, comments and recommendations are welcome at our GitHub repository: https://github.com/alan-turing-institute/the-turing-way 
+
+We would like to hear your experience in making your research reproducible and things you have learned that you think others should know! 
+Filling this form should take no longer than 5 - 10 minutes depending on the amount of detail you provide.  
+
+As these case studies will be published in the Turing Way handbook, please only include information that you are comfortable for others to read.
+
+## Tell us who you are
+> Free text answer
+
+## Which issue(s) have you encountered regarding reproducibility of your research?
+> Free text answer
+
+## Did you encounter any "gotchas" that you would like others to know about?
+Gotcha (programming): a counter-intuitive, but documented, behavior in a computer system (as opposed to a bug) [Thank you wikipedia: https://en.wikipedia.org/wiki/Gotcha]
```

Comment:
For me, thanking Wikipedia is too informal here (but I know exactly how you feel!)